src/script.babel

```diff
@@ -124,7 +130,7 @@ function textTransform() {
 	const alphabetized = alphabetize(input.value);
 	output.value = alphabetized;
 
-	if (prefixValue() === DEFAULT_PREFIX_VALUE) {
+	if (!prefixInput.value && !wordMapInput.value && !blankInput.value) {
```

Comment:
I think it's better to compare the values against their defaults, so we can intelligently show the preview if it's supported. e.g.
```suggestion
	if (prefixValue() === DEFAULT_PREFIX_VALUE && wordMapInput.value === DEFAULT_WORD_VALUE && blankInput.value === DEFAULT_BLANK_VALUE) {
```